src/otx/algorithms/segmentation/adapters/mmseg/models/heads/custom_otx_head.py

```diff
@@ -188,7 +188,7 @@ def losses(
                         seg_logit, seg_label, weight=seg_weight, ignore_index=self.ignore_index, **valid_label_mask_cfg
                     )
 
-            loss["acc_seg"] = accuracy(seg_logit, seg_label, ignore_index=self.ignore_index)
+            loss["acc_seg"] = accuracy(seg_logit.clone().detach().to("cpu"), seg_label.clone().detach().to("cpu"), ignore_index=self.ignore_index)
```

Comment:
This can be done conditionally for hpu device only to avoid any side effects when using other accelerators.

Reply:
you're right I'll update it.

Reply:
done at [e562336](https://github.com/openvinotoolkit/training_extensions/pull/2574/commits/e5623361bd91ee7471d2cb0454c9008fcea1efdb)